content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,295 @@
+cluster, you can use pod autoscaling to automatically add additional pods as needed.
+New pods are created based on an observation that a pod is exceeding its CPU
+utilization or other metric provided by the application. See
+[Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+for details.
+- *Prepare for DNS demand*: An extreme number of requests coming into the API
+server can cause excessive traffic to the DNS service. If workloads are massively
+scaling up, you will need to make sure your DNS service can scale up as well. See
+[Autoscale the DNS service in a Cluster](https://kubernetes.io/docs/tasks/administer-cluster/dns-horizontal-autoscaling/).
+- *Create additional service accounts*: While user accounts determine what users
+can do on a cluster, a service account defines pod access within a particular
+namespace in the cluster. Unless defined otherwise, a pod takes on the default
+service account for the namespace in which it runs. You can create a new service
+account, containing additional features, that you can add to a new pod to give
+it additional capabilities (see
+[Managing Service Accounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/)).
+For example, you might want to:
+  - Add secrets that a pod could use to pull images from a particular container registry. See [Configure Service Accounts for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for an example.
+  - Assign RBAC permissions to a service account. See [ServiceAccount permissions](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#service-account-permissions) for details.
+
+## Production Kubernetes using third-party providers
+
+Setting up and maintaining a production-quality Kubernetes cluster at scale
+can be a demanding, ongoing task. Instead of managing all that yourself,
+you can have third-party cloud and on-premises Kubernetes providers take on
+some or all of that work. Your level of control when working with a third-party
+provider can be anything from no responsibility for the cluster at all
+(serverless) to handing over management of just the control plane or nodes
+to running a cluster yourself on a third-party’s cloud or on-premises equipment.
+
+See [Turnkey Cloud Solutions](https://kubernetes.io/docs/setup/production-environment/turnkey-solutions/)
+for a list of available third-party Kubernetes providers.
+
+### Serverless
+
+It is possible to deploy containerized application workloads without managing
+a Kubernetes cluster at all. Cloud providers offer different options that allow
+you to deploy serverless workloads, where the underlying Kubernetes cluster is
+totally managed by them. Look for cloud providers that let you run Kubernetes
+workloads where you pay based on things like CPU usage, memory, and disk
+requests, along with some control plane costs. 
```

Comment:
I'd mention the “make it somebody else's problem” option earlier, either around the content that's on line 44 (at time of review), or in the introduction.

Reply:
Yes. I'll remove the whole section from here. Maybe I'll  make it into a short bullet list of ways not to have to set up a production Kubernetes completely yourself. Then save the longer explanation for a blog.

Reply:
Done.